src/sagas/repository.js

```diff
@@ -435,7 +435,7 @@ export function* pageQuery({ payload }) {
       results: response,
     });
   } catch (err) {
-    showNotification(err);
+    showNotification(err.response);
```

Comment:
With all of these err.responses, do we need to worry about err potentially being null?